osbs/core.py

```diff
@@ -53,6 +54,27 @@ def check_response(response):
         raise OsbsResponseException(message=content, status_code=response.status_code)
 
 
+def retry_on_conflict(func, sleep_seconds=0.5, max_attempts=10):
+    @wraps(func)
+    def retry(*args, **kwargs):
+        last_exception = None
+        for attempt in range(max_attempts):
+            if attempt != 0:
+                time.sleep(sleep_seconds)
+
+            try:
+                return func(*args, **kwargs)
+            except OsbsResponseException as ex:
+                if ex.status_code != httplib.CONFLICT:
+                    raise
+
+                last_exception = ex
```

Comment:
I wonder if it make sense to log the contents of the Build object so that our theory that its a 3.4 regression?